userspace/libsinsp/sinsp.h

```diff
@@ -162,6 +162,79 @@ enum class sinsp_plugin_platform {
 */
 class SINSP_PUBLIC sinsp : public capture_stats_source {
 public:
+	// Small sinsp event filter wrapper logic
```

Comment:
Since this is just relevant to the sinsp's internal logic, couldn't we just move it as a static definition in `sinsp.cpp` and hide it there?

Reply:
Makes sense!